data_processing/jobs/silver/silver-with-publish.yml

```diff
@@ -0,0 +1,548 @@
+# Starter pipeline
```

Comment:
Maybe rename this file to `silver-with-AIR.yml`, since we also have `silver-without-AIR.yml`? I think both are doing publish anyway, right?

Reply:
@satenderrathee